packages/terra-form-input/CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ ChangeLog
 
 Unreleased
 ----------
+### Breaking Changes
+* Removed `placeholder` prop.
+
+### Changed
+* Updated UpgradeGuide
```

Comment:
We should provide more detail about what the change to the upgrade guide implies. What was updated/added/changed?

Reply:
Update here https://github.com/cerner/terra-core/pull/3075/commits/ad81efd1977642f263c01c46fda0a8f65f4ea6ba.